src/steps/generic.rs

```diff
@@ -224,7 +224,18 @@ pub fn run_rustup(ctx: &ExecutionContext) -> Result<()> {
     let rustup = require("rustup")?;
 
     print_separator("rustup");
-    ctx.run_type().execute(rustup).arg("update").status_checked()
+    ctx.run_type().execute(&rustup).arg("update").status_checked()
```

Comment:
Clippy complains about this change: 

```
Run cross clippy --locked --target x86_64-linux-android --all-features -- -D warnings
info: downloading component 'rust-src'
info: installing component 'rust-src'
   Compiling topgrade v14.0.1 (/project)
error: the borrowed expression implements the required traits
   --> src/steps/generic.rs:227:28
    |
227 |     ctx.run_type().execute(&rustup).arg("update").status_checked()
    |                            ^^^^^^^ help: change this to: `rustup`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrows_for_generic_args
    = note: `-D clippy::needless-borrows-for-generic-args` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::needless_borrows_for_generic_args)]`

error: could not compile `topgrade` (bin "topgrade") due to 1 previous error
Error: Process completed with exit code 101.
```

Please revert it:

```suggestion
    ctx.run_type().execute(rustup).arg("update").status_checked()
```